dhis-2/dhis-services/dhis-service-dxf2/src/test/resources/dxf2/favorites/metadata_with_visualization.json

```diff
@@ -59,12 +59,12 @@
         "F_VIEW_UNAPPROVED_DATA",
         "F_INDICATOR_PUBLIC_ADD",
         "F_INDICATORGROUP_PUBLIC_ADD",
-        "F_USERGROUP_MANAGING_RELATIONSHIPS_VIEW",
+        "",
```

Comment:
This looks wrong to include empty strings. I assume we want to remove the entire string from the list